### DISCUSSION TITLE
[Projects Beta] Table view that lists Pull Requests instead of Issues

### DISCUSSION THREAD

**Discussion by AlexanderVitecsoftware** [community/community#34692] [Projects and Issues] [answered]
We lack an overview of associated pull requests for the project.
The overview should be similar to the list of pull requests for a given repository.

We would be pleased if it was possible to have a table view which lists pull requests instead of issues.
The optimal solution would be if we could make custom filters at the table view, such that we can define our own rules to which pull requests that are relevant for the view.

**Comment by sameercarpenter** [comment, marked as answer]
You can do it by creating a new view of the Table type and then adding filter `is:pr`, it will show only pull requests of that project.

**Reply by AlexanderVitecsoftware** [reply]
When I do that nothing shows..
I went to the individual PR and found that they can be assigned projects. I guess I need to mark all of my PR's individually to a project.

**Assigning pr's individually to a project and then filter with `is:pr` at the assigned project works..**
This introduces an unnecessary overhead to my workflow.

I've then tried to specify the repo at the Table filter like `repo:[some_repo]` but it only shows PR's that are assigned to the project for that repo.
**I do not understand why I can not filter as i want to?**

Would be much more convenient for me to be able to specify a filter for the PR's that I want to show at the Table.
Rather than adding an overhead to assign a new PR to a specific project when they are created.

**_At least_ is there a "short code" for assigning PR's to projects?**
Like the assignment of PR's to issues can be done by writing `Resolves #[some_number]`.
